lighthouse-core/lib/traces/pwmetrics-events.js

```diff
@@ -8,13 +8,13 @@
 
 const log = require('lighthouse-logger');
 
-function findValueInMetricsAuditFn(metricName, timingOrTimestamp) {
+function findValueInMetricsAuditFn(metricName) {
   return auditResults => {
     const metricsAudit = auditResults.metrics;
     if (!metricsAudit || !metricsAudit.details || !metricsAudit.details.items) return;
 
-    const values = metricsAudit.details.items.find(item => item.metricName === metricName);
-    return values && values[timingOrTimestamp];
+    const values = metricsAudit.details.items[0];
+    return values && values[metricName];
```

Comment:
this can be a TODO, but seems like we can just get rid of this whole `findValueInMetricsAuditFn` thing now. If there's no metrics audit, pwmetrics can't really do much, so it could check once at the beginning. Then all the gets are just property dereferencing with the new object layout.

(`metricsDefinitions` might still be useful, haven't looked into it)

Reply:
sure TODO added